pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -558,6 +560,17 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 				})
 			}
 
+			// Check if the interface name is unique
+			if _, networkAlreadyUsed := networkInterfaceMap[iface.Name]; networkAlreadyUsed {
+				causes = append(causes, metav1.StatusCause{
+					Type:    metav1.CauseTypeFieldValueDuplicate,
+					Message: fmt.Sprintf("Only one interface can be connected to one specific network"),
```

Comment:
It looks like the message doesn't correspond to what we actually validate for here: we validate that no two interfaces have the same name, not that they are connected to the same network.

Reply:
Yes but if they have the same name its mean they are connected to the same network. This is why I wrote this message